extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -107,6 +107,8 @@ export class PostgresOverviewPage extends DashboardPage {
 
 		const endpointsTable = this.modelView.modelBuilder.declarativeTable().withProperties<azdata.DeclarativeTableProperties>({
 			width: '100%',
+			ariaLabel: loc.serviceEndpoints,
+			ariaRole: loc.serviceEndpointsTable,
```

Comment:
Aria roles aren't readable strings - they're specific values : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

But the role shouldn't be necessary here - the table component should already know it's a table.

Reply:
I believe I understood issue wrong, roles to be added to text on screen being used as headers.